src/app/routes/utils/regex/index.test.js

```diff
@@ -544,7 +544,15 @@ describe('frontPage -> homePage migration', () => {
 
   const serviceToRoute = service => `/${service}`;
 
-  const servicesWithVariants = ['serbian', 'ukchina', 'zhongwen'];
+  const servicesWithVariants = [
```

Comment:
Lol, thanks for fixing the tests on this one, even though it won't be merged 😸

Reply:
Nope :) we don't run any of the tests. It does have to build in order for Storybook to work, but even then, preview does not care 😄